src/pages/publication/PublicationPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, useEffect, useState } from 'react';
+import { getPublication } from '../../api/publicationApi';
+import { addNotification } from '../../redux/actions/notificationActions';
+import i18n from '../../translations/i18n';
+import { useDispatch } from 'react-redux';
+import { CircularProgress } from '@material-ui/core';
+import { emptyPublication, Publication } from '../../types/publication.types';
+
+interface PublicationPageProps {
+  publicationId: string;
+}
+
+const PublicationPage: FC<PublicationPageProps> = ({ publicationId }) => {
```

Comment:
måtte subfixe med "page" for å unngå forveksling med publication. Kanskje vi burde gjøre det med andre "sider" også ?

Reply:
tja, kanskje det. blir jo mer eksplisitt at det er "startpunktet" for siden 👍